src/util/psbt/map/output.rs

```diff
@@ -125,6 +125,13 @@ impl TapTree {
         self.0
     }
 
+    /// Returns reference for the inner [`TaprootBuilder`]. The builder is guaranteed to be
+    /// finalized.
+    #[inline]
+    pub fn as_inner(&self) -> &TaprootBuilder {
```

Comment:
In f6dcd2cc56d942c33619bb526c089d171b04f9ab:

I think `as_inner' is an extremely weird name for this. Yes, as an implementation detail a `TapTree` is a finalized `TaprootBuilder`, but this doesn't necessarily make conceptual sense and I don't think we want to guarantee it.

I'd be ok with the same method named `as_builder`, or preferably `into_builder` which takes ownership.